src/app/api/s3-upload/route.ts

```diff
@@ -5,4 +5,7 @@ export const POST = route.configure({
   key(req, filename) {
     return sanitizeKey(filename).toLowerCase()
   },
+  endpoint: process.env.S3_UPLOAD_ENDPOINT,
+  // forcing path style is only necessary for providers other than AWS
+  forcePathStyle: !!process.env.S3_UPLOAD_ENDPOINT,
```

Comment:
Prefer using `env` from `@/lib/env` when on the server, as you get type safe values 😉 

```ts
  endpoint: env.S3_UPLOAD_ENDPOINT,
  // forcing path style is only necessary for providers other than AWS
  forcePathStyle: !!env.S3_UPLOAD_ENDPOINT,
```

Reply:
Thank you! I am now using the typed environment variables. I also resolved the merge conflicts introduced by the recent changes on `main`.